src/v2/view-builder/components/HeaderBeacon.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
I would avoid using this disable if at all possible. You can use line breaks in the template.